src/Printer/Console.php

```diff
@@ -45,7 +45,7 @@ public function printData(OutputInterface $output, FileReportList $fileReportLis
                     if (false === empty($hints)) {
                         $output->writeln('Suggestions:');
                         foreach ($hints as $hint) {
-                            $output->writeln(str_repeat(' ', 2 * self::TAB) . $hint);
+                            $output->writeln(str_repeat(' ', self::TAB) . $hint);
```

Comment:
Not sure why it doubles empty strings. IMO 4 is more than enough.
![image](https://user-images.githubusercontent.com/1302230/135719543-f454b513-622f-4bf8-85ad-7fa72e971a71.png)
against:
![image](https://user-images.githubusercontent.com/1302230/135719530-7c72755a-0056-4d25-999a-ade057d8fb70.png)

Reply:
Hi, @sidz 
Changing `src/Printer/Console.php` is duplicated #137.
Please review #137.